hack/make/binary-balena

```diff
@@ -9,8 +9,5 @@ set -e
 	export LDFLAGS="$LDFLAGS"
 	export BUILDFLAGS=( "${BUILDFLAGS[@]/netgo /}" ) # disable netgo, since we don't need it for a dynamic binary
 	source "${MAKEDIR}/.binary"
-
-	for file in balena-engine-daemon balena-engine-containerd balena-engine-containerd-shim balena-engine-containerd-ctr balena-engine-runc balena-engine-proxy; do
-		ln -sf $balena_BINARY_NAME "$DEST/$file"
-	done
+	source "${MAKEDIR}/.binary-symlinks"
```

Comment:
nit: I doubt you really need to call `source` here (the script is not supposed to set new env variables)

Reply:
will do `. "${MAKEDIR}/.binary-symlinks"` instead

Reply:
`.` is equivalent to `source`
I was thinking more about `bash "${MAKEDIR}/.binary-symlinks"`

Reply:
https://github.com/balena-os/balena-engine/blob/ci/hack/make/binary#L9 that is how upstream does it... although they also do https://github.com/balena-os/balena-engine/blob/ci/hack/test/e2e-run.sh#L71